quolance-api/src/main/java/com/quolance/quolance_api/controllers/ClientController.java

```diff
@@ -55,6 +56,19 @@ public ResponseEntity<String> deleteProject(@PathVariable(name = "projectId") Lo
         return ResponseEntity.ok("Project deleted successfully");
     }
 
+    @PutMapping("/projects/{projectId}")
+    @Operation(
+            summary = "Update a project",
+            description = "Update an existing project by passing the project details"
+    )
+    public ResponseEntity<ProjectDto> updateProject(
```

Comment:
Just a message is okay `ResponseEntity<String>`

Reply:
C better practice de return lobjet et c plus consistent avec le current design